zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/znode/groupacl/ZNodeGroupAclUtil.java

```diff
@@ -33,4 +38,48 @@
   // Has the same effect as the ZK client using ZooDefs.Ids.CREATOR_ALL_ACL.
   public static final String SET_X509_CLIENT_ID_AS_ACL =
       ZNODE_GROUP_ACL_CONFIG_PREFIX + "setX509ClientIdAsAcl";
+  // A list of domain names that will have super user privilege, separated by ","
+  public static final String SUPER_USER_DOMAIN_NAME =
+      ZNODE_GROUP_ACL_CONFIG_PREFIX + "superUserDomainName";
+  // A list of znode path prefixes, separated by ","
+  // Znode whose path starts with the defined path prefix would have open read access
+  // Meaning the znode will have (world:anyone, r) ACL
+  public static final String OPEN_READ_ACCESS_PATH_PREFIX =
+      ZNODE_GROUP_ACL_CONFIG_PREFIX + "openReadAccessPathPrefix";
+  private static Set<String> openReadAccessPathPrefixes = null;
+  private static Set<String> superUserDomainNames = null;
```

Comment:
Now this class is looking more like a Properties class. Should we just make this a `ZNodeGroupAclProperties` Singleton instead of creating individual `static` variables?

Also, this might be a place where we could consider introducing a boolean config flag that enables/disables ZNode Group-based ACL so that we can control whether this Singleton object gets initialized or not in `PrepRequestProcessor`.